server/tests/users.spec.js

```diff
@@ -0,0 +1,158 @@
+import { expect } from 'chai';
+import supertest from 'supertest';
+import app from '../app';
+
+const request = supertest(app);
+const apiEndPoint = '/api/v1/';
+const usersEndPoint = `${apiEndPoint}auth/`;
+
+describe('Users auth Tests', () => {
+  describe(`POST ${usersEndPoint}signup`, () => {
+    it('Should create a new user', (done) => {
+      const user = {
+        id: 5,
+        email: 'oluwaseun@gmail.com',
+        firstname: 'Oluwaseun',
+        lastname: 'Christopher',
+        password: 'userPW',
+        type: 'client',
+        isAdmin: false
+      };
+      request
+        .post(`${usersEndPoint}signup`)
+        .set('content-type', 'application/json')
+        .send(user)
+        .end((err, res) => {
+          if (err) return done(err);
+          expect(res.status).to.equal(201);
+          expect(res.body).to.be.an('object');
+          expect(res.body).to.have.property('data');
+          expect(res.body.data).to.have.property('token');
```

Comment:
let your tests be more specific, it needs to test for the values of these properties. Not just checking that the properties exist.

Here and the other test cases

Reply:
Got it, I will have it refactored.